backend/api_handler.py

```diff
@@ -38,6 +41,9 @@
 Worker.queue = SqsQueue.send
 
 
+MAINTENANCE_ALLOWED_OPERATIONS = ['getGroupsForUser', 'getMaintenanceWindowStatus']
```

Comment:
so this is used to allow `'getGroupsForUser', 'getMaintenanceWindowStatus'` operations when the maintenance window mode is NO-ACCESS ( in which all graphql - query and mutations - will be blocked ) so that the UI works. These are the minimally needed functions. I don't knw if there would be a case in someone wants to keep this configurable, but please let me know if you see any use case of this as such.